CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.7.4] - 2022-01-13
```

Comment:
Should we bump the MINOR version here since I don't think this qualifies as a bugfix which would require us to bump the PATCH version.

Reply:
Well I wasn’t sure as it’s a very small change in the end. No problem to change version to 0.8 :ok_hand:

Reply:
I don’t think those were right. Adding compatibility for a new Rails version is considered a feature not a bug fix from my point of view.

Reply:
No problem, I changed the version number :)